roscala/src/main/scala/coop/rchain/roscala/pools/SimpleStrandPool.scala

```diff
@@ -0,0 +1,17 @@
+package coop.rchain.roscala.pools
+import coop.rchain.roscala.GlobalEnv
+import coop.rchain.roscala.ob.Ctxt
+
+import scala.collection.mutable
+
+class SimpleStrandPool extends StrandPool {
+  private val queue = mutable.Buffer[Ctxt]()
+
+  override def append(task: (Ctxt, GlobalEnv)): Unit = queue.append(task._1)
+
+  override def prepend(task: (Ctxt, GlobalEnv)): Unit = queue.prepend(task._1)
```

Comment:
Why `(Ctxt, GlobalEnv)`?

Reply:
`ParallelStandPool` relies on `GlobalEnv` in it's logic.